zh-CN.ts

```diff
@@ -882,39 +882,39 @@ export const ZH_CN = {
     LastUpdatedAt: "最近更新于",
     DetroitBonusV3:
         "<li>发动机工厂已预先解锁</li><li>发动机工厂的产出量是原来的2倍</li><li>汽车工厂的生产能力是原来的2倍</li><li>汽车工厂使用天然气替代汽油</li><li>汽车运输免费</li><li>尼古拉.特斯拉政策免政策点</li><li>火车工厂的生产能力是原来的2倍</li>",
-    AutoScroll: "Auto Scroll",
-    SaveAndExit: "Save and Exit Game",
+    AutoScroll: "自动滚动",
+    SaveAndExit: "保存并退出游戏",
     SaveAndExitDesc:
-        "Save and exit game. This is only available on Steam version and does the same as the close button on the window bar",
-    AutoSellResourceWarningShortLabel: "💡Auto Sell",
+        "保存并退出游戏。此功能只在steam版本游戏内可用，且与直接点击窗口栏的关闭按钮作用相同。",
+    AutoSellResourceWarningShortLabel: "💡自动销售",
     AutoSellResourceWarningDesc:
-        "You are auto selling this resource, the amount stored in Trade Center cannot be used here",
-    PlayerTradeLocalPrice: "Local Price: %{price}",
-    ConstructionStatusQueueing: "%{building} is queueing for construction",
-    ConstructionStatusBuilding: "%{building} is under construction now",
-    ConstructionCancel: "Cancel Construction",
-    ConstructionStatusPaused: "%{building}'s construction is paused",
-    ChatForceScroll: "Force Scroll",
+        "你正在自动销售该资源, 保存在交易中心的该资源不能用于此处。",
+    PlayerTradeLocalPrice: "本地价格: %{price}",
+    ConstructionStatusQueueing: "%{building} 在建造队列中",
+    ConstructionStatusBuilding: "%{building} 正在建造中",
+    ConstructionCancel: "取消建造",
+    ConstructionStatusPaused: "%{building}建造暂停中",
+    ChatForceScroll: "强制滚动",
     ChatForceScrollDesc:
-        "If force scroll is on, the chat will scroll whenever there's a new message. Otherwise it will only scroll when you are already at the bottom",
-    ResourceExplorer2: "Resource Explorer 2.0",
-    ResourceExplorer2Desc: "Resource explorers produce 2x output but use 2x power",
-    ConstructionPause: "Pause",
-    ConstructionResume: "Resume",
-    PolicyBlueprint: "Blueprint",
+        "如果强制滚动开启了,聊天窗口在每收到一个新消息时都会滚动. 否则其只会在你处于聊天窗口最近消息时才会滚动。",
+    ResourceExplorer2: "资源勘探者 2.0",
+    ResourceExplorer2Desc: "资源勘探者拥有 2x 产出，但使用 2x 电力",
+    ConstructionPause: "暂停",
+    ConstructionResume: "恢复",
+    PolicyBlueprint: "蓝图模式",
     PolicyBlueprintDesc:
-        "Construction of a building will not automatically start - it has to be started manually (FREE POLICY)",
-    HighlightUnderConstruction: "...that haven't been constructed",
-    HighlightUnderLevel10: "...that are under level 10",
-    HighlightUnderLevel20: "...that are under level 20",
-    HighlightUnderLevel30: "...that are under level 30",
-    Logout: "You Are Logged Out",
-    LogoutDesc: "You have logged in on another device, this device are logged out",
-    ConstructionStatusQueueingV2: "Queueing",
-    ConstructionStatusPausedV2: "Paused",
-    ConstructionStatusBuildingV2: "Building",
-    ConstructionStatusUnpaidV2: "Unpaid",
-    ConstructionStatus: "Construction Status",
-    ConstructionStart: "Start",
-    CostFree: "Free",
+        "建筑的建造不会自动开始 - 必须手动开始 (免费政策)",
+    HighlightUnderConstruction: "...未建造过的建筑",
```

Comment:
Correction and consistency
```suggestion
    HighlightUnderConstruction: "未建造完成的...",
```